drivers/SPI.h

```diff
@@ -131,6 +131,32 @@ class SPI : private NonCopyable<SPI> {
      */
     SPI(PinName mosi, PinName miso, PinName sclk, PinName ssel, use_gpio_ssel_t);
 
+    /** Create a SPI master connected to the specified pins.
+     *
+     *  @note This constructor passes the SSEL pin selection to the target HAL.
+     *  Not all targets support SSEL, so this cannot be relied on in portable code.
+     *  Portable code should use the alternative constructor that uses GPIO
+     *  for SSEL.
+     *
+     *  @note You can specify mosi or miso as NC if not used.
+     *
+     *  @param explicit_pinmap pointer to strucure which holds static pinmap.
+     */
+    SPI(explicit_pinmap_t *explicit_pinmap);
```

Comment:
const...?

Reply:
I'm also wondering if this might better be passed as a const reference, actually. Enforces non-nullness, and potentially helps the compiler optimise a bit.